### PR TITLE
Send Content-Length header when available

### PR DESCRIPTION
**Pull request by exdis**

### CODE REVIEW

File: bin/data [hunk 1]
```diff
@@ -50,7 +50,7 @@ const command = clap.command('data', '[config] [model]')
                     return;
                 }
 
-                process.send('stream');
+                process.send(JSON.stringify({ message: 'stream', size }));
```
Comment (lahmatiy) [inline bin/data]: No need for `JSON.stringify()` here

File: lib/data.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const fs = require('fs');
-const { stringifyStream: createJsonStringifyStream } = require('@discoveryjs/json-ext');
+const { info, stringifyStream: createJsonStringifyStream } = require('@discoveryjs/json-ext');
```
Comment (lahmatiy) [inline lib/data.js]: May be `info: jsonInfo`?

File: bin/data (hunk 1 above)
Comment (lahmatiy) [inline bin/data]: `message` -> `payload` or `type`?

Comment (exdis) [reply bin/data]: fixed

Comment (exdis) [reply bin/data]: fixed

File: lib/data.js (hunk 2 above)
Comment (exdis) [reply lib/data.js]: fixed

**Pull request merged**